scripts/derivechangeset

```diff
@@ -0,0 +1,7 @@
+
+# HootApiDb requires a map layer, though (hence, 'blah').
+step1:
+	hoot derive-changeset $(HOOT_OPTS) -D convert.bounding.box=$(aoi) -D osm.changeset.sql.writer.generate.new.ids=false $(input1) $(input2) $(output) $(jobid) $(DB_URL)/blah
+
```

Comment:
I see, this is to conform with how core writes to the job status table.  But in the holy grail workflow case, input 1 is an osm api db and input 2 is a merged dataset map db.  But if the inputs were two osm files you'd need the last param?
So if I ran derive changeset from the commandline, I would need an services api db to write job status to?  This seems undesirable.  Does building --with-services enable this feature or have we broken the ability of hoot core commandline to run independent of hoot-services?